src/libs/actions/User.js

```diff
@@ -467,7 +475,8 @@ function deletePaypalMeAddress() {
             value: {},
```

Comment:
I think maybe it's not required based on what I see in the backend here:

https://github.com/Expensify/Web-Expensify/blob/290912ad1aaff2836ca4ce595f9a2c882e7bed67/lib/BankAccountAPI.php#L2803-L2811

?

Reply:
Both Paypal Onyx keys are cleared (set to nothing) in the API side but only `PAYPAL` is sent back and updated through Onyx in that snippet.

We need that `successData` to clear `NVP_PAYPAL_ME_ADDRESS` since only `PAYPAL` is being updated by the API.

Before submitting the P.R. I searched for usages of `NVP_PAYPAL_ME_ADDRESS` to understand why aren't both keys sent back through Onyx in the API but I don't think there is a reason.

I think we could and should eventually clear that in Web-E but I didn't want to spin up another P.R. for this small fix.

Does that make sense @marcaaron?

Reply:
@PauloGasparSv Should we also add `failureData`?

Reply:
I was also wondering that @sobitneupane but since it didn't have `failureData` before I think that's ok : )

Also, if an error happens I'm pretty sure they can add a new Paypal account and that would overwrite the previous + exiting the payments page and going back will re-load the Paypal account there in case it failed to delete so worst case scenario isn't that bad and IMO we can leave it like that

Reply:
I'll create a Web-E P.R. for that and put this on hold then!

Then I'll update this P.R. so it doesn't have that success data : )

cc @sobitneupane

Reply:
Oh haha nice even better :D

Reply:
@PauloGasparSv `successData` might be required to reset `pendingAction` for `ONYXKEYS.PAYPAL`

```
const successData = [
        {
            onyxMethod: CONST.ONYX.METHOD.SET,
            key: ONYXKEYS.PAYPAL,
            value: {},
        },
```

Issue found in the PR: On small screen, Paypal.me option is missing even after deleting Paypal.me.

https://user-images.githubusercontent.com/25876548/234845107-6f26f4f8-da78-434e-994f-87f8f2e86b1e.mov

Reply:
Nice catch @sobitneupane thks so much :D
Will try to figure out what's happening

Reply:
Had some time to investigate this, sadly I have no idea why this doesn't work for Android (could only replicate it in Android so far).

Both Paypal keys are being cleared in the API and if I debug with breakpoints the bug doesn't usually happen.
I think this may be related to `reconnectApp` running at the same time maybe?

I won't waste more time on this so I'm re-adding the successData here with a comment : )

Reply:
Hey, I think the android bug is universal but was easily reproducible on Android. Seems like some kind of race condition and the successData fix only helped reduce the occurrences of that. We have this issue https://github.com/Expensify/App/issues/22064 where we are looking for a solution for that. So far we think the bug is due to the use of mixed method merge and set methods.